templates/BackpackText.njk

```diff
@@ -25,6 +25,7 @@
   <style name="{{text.name}}">
     <item name="android:fontFamily">{{text.fontFamily}}</item>
     <item name="android:textSize">{{text.size}}sp</item>
+    <item name="android:letterSpacing">{{text.letterSpacing}}</item>
```

Comment:
Shouldn't this go in a separate PR?

Reply:
This is the part that generates the type scale values. CI would fail if not done along with this PR.